py3-pycurl.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: py3-pycurl
+  version: 7.45.2
+  epoch: 0
+  description: PycURL -- A Python Interface To The cURL library
+  copyright:
+    - license: LGPL/MIT
+  dependencies:
+    runtime:
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - python-3-dev
+      - py3-setuptools
+      - curl-dev
+      - openssl-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 5730590be0271364a5bddd9e245c9cc0fb710c4cbacbdd95264a3122d23224ca
+      uri: https://files.pythonhosted.org/packages/a8/af/24d3acfa76b867dbd8f1166853c18eefc890fc5da03a48672b38ea77ddae/pycurl-${{package.version}}.tar.gz
```

Comment:
Is there a reason we're not pulling this from git and polling for new tags?
- https://github.com/pycurl/pycurl